assets/javascripts/connectors/user-custom-preferences/region.js.es6

```diff
@@ -0,0 +1,452 @@
+import { action } from "@ember/object";
+
+const TIME_ZONE_TO_REGION = {
```

Comment:
How did you generate this list?

Reply:
I generated the list from [another one](https://github.com/moment/moment-timezone/blob/develop/data/meta/latest.json). Technically, I think we could update moment.js to include all the data we need.

Reply:
At the very least we could have a rake task that would update it?

Reply:
Yes do not use `.js.erb` - we have a rake task in core `javascript:update_constants` - can a plugin add to that?

Reply:
Yes, a plugin can do that. Simply by defining another rake task `javascript:update_constants` will concatenate to the existent one.